api/src/routes/edit.ts

```diff
@@ -81,6 +82,15 @@ async function getChildren(req, res) {
     res.json(response);
 }
 
+edit.get("/object/details/:pid", requireToken, pidSanitizer, async function (req, res) {
+    try {
+        const data = await HierarchyCollector.getInstance().getFedoraData(req.params.pid);
```

Comment:
HierarchyCollector is really a bad name for this service. We should probably rename it to something like FedoraLookup. Again, probably a problem to solve in the future... but just mentioning it because it's starting to bug me.